### DISCUSSION TITLE
Prepare statement lifespan

### DISCUSSION THREAD

**Discussion by steida** [rhashimoto/wa-sqlite#56] [Q&A] [answered]
Naively I think a prepared statement lifespan is a mutation, but if my app uses a few statements all the time, wouldn't be better to enlarge the lifespan to app start/end? Thank you

**Comment by rhashimoto** [comment, marked as answer]
The lifetime of a prepared statement can be whatever works for you. This is really a SQLite question and not a wa-sqlite question.

**Comment by steida** [comment]
For the record, https://www.sqlite.org/c3ref/stmt.html
